src/Resources/config/services_export_json.yml

```diff
@@ -35,3 +35,12 @@ services:
             - "@sylius.exporters_transformer_pool"
         tags:
             - { name: sylius.exporter, type: sylius.taxon, format: json }
+
+    sylius.exporter.tax_categories.csv:
+        class: FriendsOfSylius\SyliusImportExportPlugin\Exporter\JsonResourceExporter
+        arguments:
+            - "@sylius.exporter.pluginpool.tax_categories"
+            - ["Id", "Code" ,"Name", "Description"]
```

Comment:
I'd change `Name` and `Description` for `Translations`, it's done this way in #5 so we can actually export all the translated names and descriptions.
`Id` is not needed here also since we've got code.